docs/schema.json

```diff
@@ -2904,7 +2923,8 @@
         },
         "loanConsent": {
```

Comment:
naming: 1) this is Concent but output key doesn't fin into expected naming pattern (expected "loan", but loan is taken for something else)